### ISSUE TITLE
What is the role of MultiLabelMarginLoss()

### ISSUE THREAD

**Issue by d049** [poojahira/image-captioning-bottom-up-top-down#5]

**Comment by poojahira** [comment]
An additional loss function, for discriminative loss. I got the idea from a few papers that are using this for image captioning, in addition to cross entropy loss.
